src/app/components/stacks-transaction-item/increase-fee-button.tsx

```diff
@@ -31,7 +31,7 @@ export function IncreaseFeeButton(props: IncreaseFeeButtonProps) {
       zIndex={999}
     >
       <HStack gap="space.00">
-        <ChevronsRightIcon color="stacks" mr="3px" />
+        <ChevronsRightIcon color="stacks" mr="3px" variant="small" />
```

Comment:
Maybe we can use tokens for some of these `px` values in future? `3px` seems like a weird margin

Reply:
Thinking this is wrong and I'll just change it to `space.01`.